cogs/polling.py

```diff
@@ -0,0 +1,124 @@
+import discord
+import asyncio
+from discord.ext import commands
+import random
+from discord.ext.commands.cooldowns import BucketType
+
+
```

Comment:
Missing doc comment!

```suggestion
    # -----------------------------------------------------------------------------------------------------------------
    #    Function: poll(self, ctx)
    #    Description: TODO
    #    Inputs:
    #       - ctx: context of the command
    #       -
    #    Outputs:
    #       - TODO
    # -----------------------------------------------------------------------------------------------------------------
```